x-pack/plugins/uptime/public/components/queries/ping_list/ping_list.tsx

```diff
@@ -71,7 +71,7 @@ export class Pings extends React.Component<Props, PingListState> {
     this.state = {
       statusOptions,
       selectedOption: statusOptions[2],
-      maxSearchSize: 200,
+      maxSearchSize: 100,
```

Comment:
Since this value exists in multiple places in the file, it's occurring to me that we should maintain it in a constant. Probably in the file above the class definition is fine, or in the `~/common` folder.